ecs-agent/netlib/network_builder.go

```diff
@@ -0,0 +1,46 @@
+package netlib
```

Comment:
nit & non-blocking comment
1. Doc comments are missing in new files
```
// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
//
// Licensed under the Apache License, Version 2.0 (the "License"). You may
// not use this file except in compliance with the License. A copy of the
// License is located at
//
//	http://aws.amazon.com/apache2.0/
//
// or in the "license" file accompanying this file. This file is distributed
// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
// express or implied. See the License for the specific language governing
// permissions and limitations under the License.
``` 
2. We can add log messages to help debug. Find examples as follows.
* Info level log: [here](https://github.com/aws/amazon-ecs-agent/blob/master/ecs-agent/modeltransformer/transformer.go#L68C4-L68C15)
* Error level log: [here](https://github.com/aws/amazon-ecs-agent/blob/master/ecs-agent/acs/session/attach_task_eni_responder.go#L116)
* Warn level log: [here](https://github.com/aws/amazon-ecs-agent/blob/master/ecs-agent/acs/session/attach_task_eni_responder.go#L89)
* Debug level log: [here](https://github.com/aws/amazon-ecs-agent/blob/master/ecs-agent/acs/session/attach_resource_responder.go#L89)

Reply:
Thank @chienhanlin for calling that out. This PR is one of the first in a list changes for UNL. I have already put in place the copyright comments for the missing pages in the next commit and I will make sure the required logging is also put in place.